pandas/tests/plotting/test_backend.py

```diff
@@ -45,40 +44,28 @@ def test_backend_can_be_set_in_plot_call(monkeypatch, restore_backend):
     assert df.plot(backend="pandas_dummy_backend") == "used_dummy"
 
 
-@td.skip_if_no_mpl
-def test_register_entrypoint(restore_backend):
-
-    dist = pkg_resources.get_distribution("pandas")
```

Comment:
this test was invalid because the underlying code uses importlib.metadata and not pkg_resources - however it passed because the entrypoint name matched the module name and so the code fell back to a regular import